src/main/java/io/tackle/controls/entities/StakeholderGroup.java

```diff
@@ -30,18 +26,15 @@
     @Filterable
     public String description;
 
-    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @ManyToMany(fetch = FetchType.LAZY)
     @JoinTable(
             name = "stakeholdergroup_stakeholders",
             joinColumns = {@JoinColumn(name = "group_id")},
             inverseJoinColumns = {@JoinColumn(name = "stakeholder_id")}
     )
-    @JsonManagedReference
```

Comment:
Just so I understand, is this not needed because we've removed the cascade?

Reply:
To avoid Jackson's "circular marshalling" I introduced the `@JsonBackReference` annotation in `Stakeholder` and it should have made this annotation not needed anymore.